### PR TITLE
NAS-103131: Begin updating 'Dynamic DNS' docs section

### PR DESCRIPTION
**Pull request by Mrt134**
- Review and condense 'DDNS Configuration Options' table.
- Add entries for missing fields and fill with placeholder help text. Still investigating improvements to the text.
- HTML build test: no issues.

TODO: Finish investigating the Custom fields to improve the help text, retest the error behavior and update docs accordingly, and sync help text with webui.

### CODE REVIEW

File: userguide/services.rst [hunk 1]
```diff
@@ -224,41 +224,35 @@ service in :menuselection:`Services --> Dynamic DNS`.
...
-   |                       |                   |                                                                                                                 |
    +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
-   | Use SSL               | checkbox          | Set to use HTTPS for the connection to the server that updates the DNS record.                                  |
-   |                       |                   |                                                                                                                 |
+   | SSL                   | checkbox          | Set to use HTTPS for the connection to the server that updates the DNS record.                                  |
+   +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
+   | Custom Server         | string            | Hostname for your custom DDNS provider.                                                                         |
```
Comment (wblock) [inline userguide/services.rst]:
```suggestion
   | Custom Server         | string            | Hostname for the custom DDNS provider.                                                                          |
```

**Comment by Mrt134** [comment]
Investigation: looks like https://github.com/troglobit/inadyn#custom-ddns-providers has the most relevant information about these fields

File: userguide/services.rst [hunk 2]
```diff
@@ -224,41 +224,41 @@ service in :menuselection:`Services --> Dynamic DNS`.
...
+   | CheckIP Server        | string            | Enter the name and port of the server that reports the external IP address. For example, entering               |
+   |                       |                   | :samp:`checkip.dyndns.org:80` uses `Dyn IP detection <https://help.dyn.com/remote-access-api/checkip-tool/>`__  |
+   |                       |                   | to discover the remote socket IP address.                                                                       |
    +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
-   | CheckIP Path          | string            | Enter the path that is requested by the *CheckIP Server* to determine the user IP address.                      |
-   |                       |                   |                                                                                                                 |
+   | CheckIP Path          | string            | The path to the :guilabel:`CheckIP Server`. For example, the *no-ip.com* provider uses the                      |
```
Comment (wblock) [inline userguide/services.rst]:
```suggestion
   | CheckIP Path          | string            | Path to the :guilabel:`CheckIP Server`. For example, the *no-ip.com* provider uses the                          |
```

File: userguide/services.rst [hunk 3]
```diff
@@ -224,41 +224,41 @@ service in :menuselection:`Services --> Dynamic DNS`.
...
+   |                       |                   | :samp:`/checkip.html` path for the :samp:`dynamic.zoneedit.com` :guilabel:`CheckIP Server`.                     |
```
Comment (wblock) [inline userguide/services.rst]: The multiple clauses in the example make it kind of hard to tell which value is entered where.  Does it mean that for no-ip.com, CheckIP Server is set to dynamic.zoneedit.com and CheckIP Path is set to /checkip.html?

**Comment by Mrt134** [comment]
I reviewed again it looks ready to merge.

**Pull request merged**